webapp/views/map/script.js

```diff
@@ -108,19 +32,20 @@ const baseLayers = {
   "OSM Standard style": osm,
   "OSM Humanitarian style": hot
 }
+
 const groupedOverlays = {
   "Basemap": {
-    "Waterways": waterways,
-    "Roads": roads,
-    "Buildings": buildings,
-    "Basemap boundary": basemapBbox,
-    "Current selection": selection
+    [services.waterways.displayName]: createWms(services.waterways),
+    [services.roads.displayName]: createWms(services.roads),
+    [services.buildings.displayName]: createWms(services.buildings),
+    [services.basemapBbox.displayName]: createWms(services.basemapBbox),
+    [services.selection.displayName]: createWms(services.selection)
```

Comment:
This should be a one-liner!

Reply:
Moved grouping logic into config in the newest commit